http_status/lib/rfc7231.status_codes

```diff
@@ -33,6 +33,7 @@ Code Reason-Phrase
 415 Unsupported Media Type
 416 Range Not Satisfiable
 417 Expectation Failed
+422 Unprocessable Entity
```

Comment:
RFC 7231 changed the definition of Bad Request. It used to be only used for things like invalid JSON, malformed headers etc. Now it can be applied to anything in general that the client did wrong.